### PR TITLE
Unobscure scroll bar in questDetail component (quest details participant list)

### LINKED ISSUE #9537: scroll bar mostly hidden in quest details participants list
From Aerilita ( 3b724a41-48d3-4048-ae27-70b258cf5a72 ) in the Report a Bug guild:

"On the website in quest details the list of participants is positioned in such a way that its scrollbar is mostly hidden by the quest description in the middle. (I could scroll it with the mouse wheel though.)"
[to illustrate](https://pasteboard.co/GTWWdtC.gif)

I've confirmed that. In fact it's so well hidden on my browsers I didn't realise there was meant to be a visible scroll bar. :)

### PR DESCRIPTION
**Pull request by pcvonz**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #9537

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

- Adjust `left-panel` positioning in the `questDetailModal` component so the scroll-bar is not obscured.
- Change `overflow` of the left-panel to `overflow-y`. I added an extra long username to the left-panel and it wrapped to the next line, so I assume `overflow-x` isn't necessary. 

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID: b16e0c46-c20b-4aea-9f1f-6e9dd80ad61b

### CODE REVIEW

File: website/client/components/groups/questDetailsModal.vue [hunk 1]
```diff
@@ -52,10 +52,10 @@
     height: 460px;
     width: 320px;
     top: 2.5em;
-    left: -22em;
+    left: -22.8em;
     z-index: -1;
     padding: 2em;
-    overflow: scroll;
+    overflow-y: scroll;
```
Comment (paglias) [inline website/client/components/groups/questDetailsModal.vue]: can we get it to work with overflow-y: auto?

**Comment by paglias** [comment]
@pcvonz thanks for this PR! It'll be merged next week

**Comment by pcvonz** [comment]
Changed `overflow-y: scroll` to `overflow-y: auto`.

**Comment by SabreCat** [comment]
Thank you @pcvonz, and welcome to the ranks of the Blacksmiths! Note that contributor tiers require increasing amounts of effort from one to the next, so tier 2 may require multiple PRs or a larger PR to attain. But keep helping out and we'll express our gratitude accordingly!

**Comment by joedragons** [comment]
That is a great label (system) for wondering randoms=)  Thanks.

**Pull request merged**